Source/Core/Common/MemoryUtil.cpp

```diff
@@ -34,39 +34,117 @@
 namespace Common
 {
 
-#if !defined(_WIN32) && defined(_M_X86_64) && !defined(MAP_32BIT)
+//#if !defined(_WIN32) && defined(_M_X86_64) && !defined(MAP_32BIT)
```

Comment:
need to prevent this on Windows since unistd isn't a thing there.

Reply:
or move the include down to the next ifdef i guess

Reply:
Fixed!